R/sensitivityCalculation.R

```diff
@@ -22,10 +22,7 @@
 #'   spreadsheet.
 #'
 #' @examples
-#'
-#' # for reproducibility
-#' set.seed(123)
-#'
+#' \dontrun{
```

Comment:
Running those examples take too much time

Reply:
You can also skip the examples in plotting functions for sensitivity calculation. 

We have tests for these, so running examples is not crucial.